docs/data/material/getting-started/templates/blog/NavBar.js

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { createTheme, ThemeProvider, styled } from '@mui/material/styles';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import Box from '@mui/material/Box';
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Container from '@mui/material/Container';
+import ArrowBackRoundedIcon from '@mui/icons-material/ArrowBackRounded';
+import ToggleColorMode from './components/ToggleColorMode';
+import getBlogTheme from './theme/getBlogTheme';
+
+const StyledAppBar = styled(AppBar)(({ theme }) => ({
+  position: 'fixed',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'space-between',
+  flexShrink: 0,
+  borderBottom: '1px solid',
+  borderColor: theme.palette.divider,
+  backgroundColor: theme.palette.background.paper,
+  boxShadow: theme.shadows[1],
+  backgroundImage: 'none',
+  padding: 4,
+}));
+
+function NavBar({ showCustomTheme, toggleCustomTheme, mode, toggleColorMode }) {
+  const handleChange = (event) => {
+    toggleCustomTheme(event.target.value === 'custom');
+  };
+  const blogTheme = createTheme(getBlogTheme(mode));
+
+  return (
+    <ThemeProvider theme={blogTheme}>
+      <StyledAppBar>
+        <Container maxWidth="lg">
+          <Toolbar
+            variant="dense"
+            disableGutters
+            sx={{ display: 'flex', justifyContent: 'space-between' }}
+          >
+            <Button
+              variant="text"
+              size="small"
+              aria-label="Back to templates"
+              startIcon={<ArrowBackRoundedIcon />}
+              component="a"
+              href="/material-ui/getting-started/templates/"
+              sx={{ display: { xs: 'none', sm: 'flex' } }}
+            >
+              Back to templates
+            </Button>
+            <IconButton
+              size="small"
+              aria-label="Back to templates"
+              component="a"
+              href="/material-ui/getting-started/templates/"
+              sx={{ display: { xs: 'auto', sm: 'none' } }}
+            >
+              <ArrowBackRoundedIcon />
+            </IconButton>
+            <Box sx={{ display: 'flex', gap: 1 }}>
+              <FormControl variant="outlined" sx={{ minWidth: 180 }}>
+                <Select
+                  size="small"
+                  labelId="theme-select-label"
```

Comment:
Those don't exist:

<img width="953" alt="SCR-20240921-pdym" src="https://github.com/user-attachments/assets/e553c2ab-dd28-4335-a09c-23cce561b72a">

https://pagespeed.web.dev/analysis/https-mui-com-material-ui-getting-started-templates-dashboard/3z1h84d300?form_factor=desktop

Reply:
Thanks for the feedback—it should be fixed in #43604